libderections-common/src/test/java/com/mapbox/navigation/route/common/NavigationRouteCallbackTest.kt

```diff
@@ -23,37 +23,41 @@ class NavigationRouteCallbackTest {
         val call = mockk<Call<DirectionsResponse>>()
         val uuid = "some_uuid"
         val response = buildMockResponse(uuid)
-        val routeCallback = NavigationRouteCallback(listener, callback)
+        val routeCallback =
+            NavigationRouteCallback(listener, callback)
 
         routeCallback.onResponse(call, response)
 
         verify { callback.onResponse(call, response) }
     }
 
-    @Test
-    fun onResponse_validResponseSendsEvent() {
-        mockkObject(NavigationTelemetry)
-        val listener = mockk<NavigationRouteEventListener>()
-        val elapsedTime = mockk<ElapsedTime>()
-        every { listener.time } returns elapsedTime
-        val callback = mockk<Callback<DirectionsResponse>>(relaxed = true)
-        val call = mockk<Call<DirectionsResponse>>()
-        val uuid = "any_uuid"
-        val response = buildMockResponse(uuid)
-        val routeCallback = NavigationRouteCallback(listener, callback)
-
-        routeCallback.onResponse(call, response)
-
-        verify { NavigationTelemetry.routeRetrievalEvent(eq(elapsedTime), eq(uuid)) }
-    }
+    // TODO move NavigationTelemetry to base module ?
```

Comment:
why are we commenting out this test?

Reply:
because here I need `NavigationTelemetry` for tests. And for this `NavigationTelemetry` should be moved to `base` or `metric` module